### PR TITLE
Generate metric_sender.yaml from template

### PR DESCRIPTION
**Pull request by zgalor**
Generate the metric_sender.yaml file from template and place it in /etc/openshift_tools/metric_sender.yaml

### CODE REVIEW

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 1]
```diff
@@ -13,6 +13,93 @@ oso_snapshot_aws_secret_access_key: "{{ osohm_snapshot_aws_secret_access_key }}"
...
+    pass: "{{ osohm_default_zagg_server_password }}"
+    ssl_verify: "{{ osohm_zagg_verify_ssl }}"
+    verbose: False
+    debug: False
+  hawk:
+    active: False
+    url: "{{ osohm_hawk_web_url }}"
+    user: "{{ osohm_default_hawk_server_user }}"
+    pass: "{{ osohm_default_hawk_server_password }}"
+    ssl_verify: "{{ osohm_hawk_verify_ssl }}"
```
Comment (zgalor) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: @joelddiaz I wasn't sure whether and where I need to define those "osohm_hawk_web_url", etc.
I couldn't find a definition of the "zagg" ones.

**Comment by zgalor** [comment]
@twiest @joelddiaz Could you please review? Lots of PRs depend on this one

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 (hunk 1 above)
Comment (joelddiaz) [reply ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: @zgalor vars like osohm_zagg_* are passed into this oso_host_monitoring role, and the vars are defined in our private data hierarchy

**Review by joelddiaz — CHANGES_REQUESTED** [review]
For this PR, we should add a section into oso_host_monitoring/defaults/main.yml to house the off-by-default hawk variables. Basically just copying the defaults from docker/oso-host-monitoring/src/container_setup/monitoring-config.yml:

osohm_hawk_config:
  active: False
  url: https://localhost:8443
  user: admin
  pass: password
  ssl_verify: False
  verbose: False
  debug: False

Then in oso_host_monitoring/templates/monitoring-config.yml:

hawk: "{{ osohm_hawk_config }}"

Now when we stand up a hawkular server we'll be able to override this whenever we call out to the oso_host_monitoring role.

**Comment by joelddiaz** [comment]
@zgalor probably should rebase on a recent prod b/c of changes in #1776

**Comment by zgalor** [comment]
@joelddiaz I made the changes you requested. Please check

**Review by joelddiaz — APPROVED** [review]

**Comment by joelddiaz** [comment]
@zgalor looks good. just need to resolve the merge conflict before we can merge this through

**Comment by zgalor** [comment]
@joelddiaz Done

**Pull request merged**